exporters/zipkin/include/opentelemetry/exporters/zipkin/zipkin_exporter.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include "opentelemetry/exporters/zipkin/recordable.h"
+#include "opentelemetry/ext/http/client/http_client_factory.h"
+#include "opentelemetry/ext/http/common/url_parser.h"
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include "nlohmann/json.hpp"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+/**
+ * Struct to hold OTLP exporter options.
```

Comment:
Update all the comments from OTLP to zipkin?

Reply:
haha..nice catch : )

Reply:
fixed now. thanks for noticing it.